server/filestore.go

```diff
@@ -6797,7 +6817,7 @@ var dios chan struct{}
 // golang.org's semaphore seemed a bit heavy.
 func init() {
 	// Based on Go max threads of 10k, limit ourselves to a max of 1k blocking IO calls.
-	const nIO = 1024
+	const nIO = 4
```

Comment:
Does the above comment need to be updated?

Reply:
Yep thanks.